src/modules/hip/kernel/down_mixing.hpp

```diff
@@ -0,0 +1,64 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+
+__global__ void down_mixing_hip_tensor(float *srcPtr,
+                                       uint srcStride,
+                                       float *dstPtr,
+                                       uint dstStride,
+                                       int *srcDimsTensor)
+
+{
+    int id_x = hipBlockIdx_x * hipBlockDim_x + hipThreadIdx_x;
+    int id_z = hipBlockIdx_z * hipBlockDim_z + hipThreadIdx_z;
+    int srcLength = srcDimsTensor[id_z * 2];
+    int channels = srcDimsTensor[id_z * 2 + 1];
+
+    if (id_x >= srcLength)
+        return;
+
+    // multi channel
+    if(channels > 1)
```

Comment:
Regarding https://github.com/r-abishek/rpp/pull/261/files#r1595772717
We can either split the kernels or combine them without the if statement. Pls check